api/serverpb/server.proto

```diff
@@ -137,6 +139,16 @@ message ChangeSettingsResponse {
   Settings settings = 1;
 }
 
+message SetSSHKeyRequest {
+  string ssh_key = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
```

Comment:
Why this is not a part of `ChangeSettingsRequest`?

Reply:
Because SSH key replaces all current keys each time we call it. If user adds new ssh key manually it will rewrite file.

Reply:
We have another option to check equality of current ssh key and saved ssh key. But in this case if user manually changed ssh keys file we won't be able to restore previously uploaded ssh key.

Reply:
If `ssh_key` is empty, do not replace it. That's what what we do for all settings.
Otherwise, there is no symmetry between setting a value (separate method) and getting a value (common method).

Reply:
> in this case if user manually changed ssh keys file we won't be able to restore previously uploaded ssh key

Why we don't read it from file and return in GetSettings?

Reply:
but how we will handle it on frontend side? we just moving this problem from backend side to frontend side.
We should show current key on frontend side.

Reply:
> Why we don't read it from file and return in GetSettings?
That's a good option.

Reply:
About why did we separated method. I've never seen keeping ssh keys as part of other settings. Usually it's separated.

Reply:
But we do return it as a part of Settings.